lib/web_ui/lib/src/engine/skwasm/skwasm_impl/paragraph.dart

```diff
@@ -102,8 +103,30 @@ class SkwasmParagraph implements ui.Paragraph {
   bool get didExceedMaxLines => paragraphGetDidExceedMaxLines(handle);
 
   @override
-  void layout(ui.ParagraphConstraints constraints) =>
+  void layout(ui.ParagraphConstraints constraints) {
     paragraphLayout(handle, constraints.width);
+    if (!_hasCheckedForMissingCodePoints) {
+      _hasCheckedForMissingCodePoints = true;
+      final int missingCodePointCount = paragraphGetUnresolvedCodePoints(handle, nullptr, 0);
+      if (missingCodePointCount > 0) {
+        withStackScope((StackScope scope) {
+          final Pointer<Uint32> codePointBuffer = scope.allocUint32Array(missingCodePointCount);
+          final int returnedCodePointCount = paragraphGetUnresolvedCodePoints(
+            handle,
+            codePointBuffer,
+            missingCodePointCount
+          );
+          assert(missingCodePointCount == returnedCodePointCount);
+          renderer.fontCollection.fontFallbackManager!.addMissingCodePoints(
+            List<int>.generate(
+              missingCodePointCount,
+              (int index) => codePointBuffer[index]
+            )
```

Comment:
I think probably the reason this is failing is because the `T` in `Pointer<T>` the native type, but the `T` in `List<T>` is probably the dart type (`int` in this case). That makes this kind of difficult to create a generic helper for.